testsuite/features/secondary/srv_advanced_search.feature

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_monitoring
+@sle_minion
+@sle_client
+Feature: Advanced Search
+  In order to check and maintain the clients and minions
+  As an authorized user
+  I want to be able to search for specific systems according to location or other characteristics
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: No search results - inverse results
+    When I follow the left menu "Systems > Advanced Search"
+    And I enter minion hostname on the search field
+    And I select "Hostname" from "Field to Search"
+    And I check "invertlabel"
+    And I click on "Search"
+    Then I should see a "No results found." text
+
+  Scenario: One search result for City
+    Given I have a system with "Little Whinging" as "City" property
+    When I follow the left menu "Systems > Advanced Search"
+    And I enter "Little Whinging" on the search field
+    And I select "City" from "Field to Search"
+    And I check "fineGrainedlabel"
+    And I click on "Search"
+    Then I should land on system's overview page
+
+  Scenario: One search result for State/ Province
```

Comment:
this was intentional since the scenario is normal readable english, where there's usually a space after the /

Reply:
I see. In this case you wanted to indicate an "or" don`t you?
[Grammerly](https://www.grammarly.com/blog/slash/) states the following:

> Often, when a slash is used in a formal or informal text, it is meant to indicate the word or. The examples below illustrate this meaning of the forward slash:
>
>- When leaving the classroom, the teacher noticed that a student had left his/her backpack.
>- College freshmen should bring a mattress and/or cot to sleep on during orientation.
>- If/when Mary ever shows up, we can all head out to the party together.
>- Burgers or pizza for dinner? Yeah, either/or is fine with me.

I only know the above case or to use 2 spaces, one before and one after the slash. But I guess you have more experience since you worked in the UK :wink:

Reply:
I didn't take a literature degree or anything of the sort (yet) so I'll take grammarly's word for this then and update accordingly :p TIL